llvm/lib/Target/AIE/AIEInterBlockScheduling.cpp

```diff
@@ -708,10 +712,13 @@ void InterBlockScheduling::enterRegion(MachineBasicBlock *BB,
   auto &BS = getBlockState(BB);
   DEBUG_BLOCKS(dbgs() << "    >> enterRegion, Iter=" << BS.FixPoint.NumIters
                       << "\n");
+
+  // Only add regions of loops when in the GatheringRegions phase
   if (BS.Kind != BlockType::Loop ||
       BS.FixPoint.Stage == SchedulingStage::GatheringRegions) {
-    // Only add regions of loops when in the GatheringRegions phase
-    BS.addRegion(BB, RegionBegin, RegionEnd);
+    ArrayRef<MachineBundle> TopFixedBundles;
+    ArrayRef<MachineBundle> BotFixedBundles;
```

Comment:
Uninitialized & dead?

Reply:
Those represent empty arrays, once I want non-empty "fixed" regions, they will be referencing e.g. `BlockState::BottomInsert`